### PR TITLE
Sponsorship section, for those companies who wanted to see the deck

### PR DESCRIPTION
**Pull request by stefancherubin**

### CODE REVIEW

File: src/js/modules/core/components/HomePage.tsx [hunk 1]
```diff
@@ -94,6 +100,10 @@ class HomePage extends React.Component<Props, State> {
             <img className={classes.curvedTop} src="/img/semicircle.svg" />
             <FAQ />
           </div>
+          <div className={classes.sponsorshipSection}>
+          <img className={classes.curvedTop} src="/img/semicircle.svg" />
```
Comment (NicholasLYang) [inline src/js/modules/core/components/HomePage.tsx]: We're using the curved top for both?

File: src/js/modules/core/components/SponsorshipSection.tsx [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+  },
+  link: {
+    color: "yellow",
+    "&:hover": {
+      color: "white"
+    }
+  }
+});
+
+const SponsorshipInfo: React.SFC<Props> = ({ classes }) => {
```
Comment (NicholasLYang) [inline src/js/modules/core/components/SponsorshipSection.tsx]: Shouldn't this be SponsorshipSection?

**Comment by themichaelyang** [comment]
Doesn't seem to be scaling correctly locally: 
screen shot 2019-01-16 at 8 53 07 am

**Comment by themichaelyang** [comment]
Also: @kristelfung suggested we use a white background, so we don't run into any issues w/ colored logos. Thoughts? @stefancherubin

**Comment by stefancherubin** [comment]
Yea, some logos get smaller if they are stacked next to each other, JPMC for example. I'm fine with using a white background

**Comment by themichaelyang** [comment]
I think the solution to the scaling issue is to nest the images in divs. Set the divs to have `flex: 1` and the images to `max-width: 100%`.

**Comment by NicholasLYang** [comment]
Idk if this helps, but I've used svgr in the past to make components from the logos https://github.com/smooth-code/svgr

**Pull request closed without merging**